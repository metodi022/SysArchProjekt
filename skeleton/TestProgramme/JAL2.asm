lui $1, 0x1234
jal test1
test2:
jal end
test1:
jal test2
end: