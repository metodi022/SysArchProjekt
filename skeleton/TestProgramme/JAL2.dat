0x3c011234
0x0c100003
0x0c100004
0x0c100002
