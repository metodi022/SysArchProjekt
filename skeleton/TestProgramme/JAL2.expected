@1 12340000
@1F 0040000c